frontend/src/app/testQueue/AoEForm/SymptomInputs.tsx

```diff
@@ -70,21 +70,33 @@ const SymptomInputs: React.FC<Props> = ({
         )}
       </div>
       {!noSymptoms && (
-        <TextInput
-          type="date"
-          label="Date of symptom onset"
-          name="symptom_onset"
-          value={onsetDate}
-          onChange={(e) => setOnsetDate(e.target.value)}
-          min="2020-02-01"
-          max={new Date().toISOString().split("T")[0]}
-          required={Object.keys(currentSymptoms).some(
-            (key) => currentSymptoms[key]
+        <>
+          <Label htmlFor="meeting-time">Date of symptom onset</Label>
+          <span className="usa-hint">mm/dd/yyyy</span>
+          {symptomOnsetError === "error" && (
+            <span className="usa-error-message" role="alert">
+              <span className="usa-sr-only">Error: </span>
+              {symptomOnsetError}
+            </span>
```

Comment:
I think `symptomOnsetError` is the actual description of the error rather than literally the string `"error"`. The following should work, I think

```suggestion
          {symptomOnsetError && (
            <span className="usa-error-message" role="alert">
              <span className="usa-sr-only">Error: </span>
              {symptomOnsetError}
            </span>
```